src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -667,14 +668,64 @@ def generate_pvt_model_compositional(value: DescriptionDocument):
     }
 
 
+def load_combined_fluid_description(
+    document: DescriptionDocument,
+) -> Dict[str, case_description.CombinedFluidDescription]:
+    alfacase_to_case_description = {
+        "pvt_model": load_value,
+    }
+
+    def generate_fluid_description(
+        value: DescriptionDocument,
+    ) -> case_description.CombinedFluidDescription:
+        case_values = to_case_values(value, alfacase_to_case_description)
+        item_description = case_description.CombinedFluidDescription(**case_values)
+        return update_multi_input_flags(document, item_description)
+
+    return {
+        key.data: generate_fluid_description(
+            DescriptionDocument(value, document.file_path)
+        )
+        for key, value in document.content.items()
+    }
+
+
+def load_pvt_model_combined_description(
+    document: DescriptionDocument,
+) -> Dict[str, case_description.PvtModelCombinedDescription]:
+    alfacase_to_case_description = {
+        "reference_pvt_model": load_value,
+        "fluids": load_combined_fluid_description,
+    }
+
+    def generate_pvt_model_combined(value: DescriptionDocument):
+        case_values = to_case_values(value, alfacase_to_case_description)
+        item_description = case_description.PvtModelCombinedDescription(**case_values)
+        return update_multi_input_flags(document, item_description)
+
+    return {
+        key.data: generate_pvt_model_combined(
+            DescriptionDocument(value, document.file_path)
+        )
+        for key, value in document.content.items()
+    }
+
+
 def load_pvt_models_description(
     document: DescriptionDocument,
 ) -> case_description.PvtModelsDescription:
     alfacase_to_case_description = {
```

Comment:
This could be changed by a call to `load_instance`, the problem is `"tables": load_pvt_tables,`, that function do not have support for explict loaders. I remember suggesting someone to add that but he did not...

You could do:

```python
def load_pvt_models_description(
    document: DescriptionDocument,
) -> case_description.PvtModelsDescription:
    alfacase_to_case_description = get_case_description_attribute_loader_dict(
        case_description.PvtModelsDescription, explicit_loaders={"tables": load_pvt_tables}
    )
    case_values = to_case_values(document, alfacase_to_case_description)
    item_description = class_(**case_values)
    return update_multi_input_flags(document, item_description)
```

Or do:

```diff
diff --git a/src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py b/src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py
index b857195..ab63dd6 100644
--- a/src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py
+++ b/src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py
@@ -119,11 +119,17 @@ def update_multi_input_flags(document: DescriptionDocument, item_description: T)
     return item_description
 
 
-def load_instance(alfacase_content: DescriptionDocument, class_: Type[T]) -> T:
+def load_instance(
+    alfacase_content: DescriptionDocument,
+    class_: Type[T],
+    explicit_loaders: Optional[Dict[str, Callable]] = None,
+) -> T:
     """
     Create an instance of class_ with the attributes found in alfacase_content.
     """
-    alfacase_to_case_description = get_case_description_attribute_loader_dict(class_)
+    alfacase_to_case_description = get_case_description_attribute_loader_dict(
+        class_, explicit_loaders=explicit_loaders
+    )
     case_values = to_case_values(alfacase_content, alfacase_to_case_description)
     item_description = class_(**case_values)
     return update_multi_input_flags(alfacase_content, item_description)
@@ -654,15 +660,11 @@ def load_pvt_model_compositional_description(
 def load_pvt_models_description(
     document: DescriptionDocument,
 ) -> case_description.PvtModelsDescription:
-    alfacase_to_case_description = {
-        "default_model": load_value,
-        "tables": load_pvt_tables,
-        "correlations": load_pvt_model_correlation_description,
-        "compositions": load_pvt_model_compositional_description,
-    }
-    case_values = to_case_values(document, alfacase_to_case_description)
-    item_description = case_description.PvtModelsDescription(**case_values)
-    return update_multi_input_flags(document, item_description)
+    return load_instance(
+        document,
+        case_description.PvtModelsDescription,
+        explicit_loaders={"tables": load_pvt_tables},
+    )
 
 
 # Used for testing - Will be ignored on the document contents
```

Reply:
BTW, this is not a blocking issue for my approval =)

Reply:
Oh my, this is awesome. Thank you!